pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TopicTransactionBuffer.java

```diff
@@ -245,6 +248,20 @@ public CompletableFuture<Void> checkIfTBRecoverCompletely(boolean isTxnEnabled)
         }
     }
 
+    @Override
+    public long getOngoingTxnCount() {
```

Comment:
IIUC `ongoingTxns` and `aborts` are real gauges because they can be increased or decreased. 
`txnCommittedCounter` is only incremented and so it's a counter. 
Is it correct ? 
If so, I believe we have to introduce a txnAbortedCounter

Reply:
it makes sense.